src/dotnet/commands/dotnet-publish/LocalizableStrings.resx

```diff
@@ -141,4 +141,7 @@
   <data name="SelfContainedOptionDescription" xml:space="preserve">
     <value>Publish the .NET Core runtime with your application so the runtime doesn't need to be installed on the target machine. Defaults to 'true' if a runtime identifier is specified.</value>
   </data>
+  <data name="NoBuildOptionDescription" xml:space="preserve">
+    <value>Do not build project before publishing.  Implies --no-restore.</value>
```

Comment:
Should this say `Do not build the project before publishing. Implies --no-restore.`? Notice the `the` before `project`

Reply:
This is copied from pack with pack replaced by publish. I think the current wording is fine, but I can change both if we have consensus. I do think publish and pack should match.

@mairaw @kathleendollard Can you make a call. I won't block on this and will happily send a followup PR if I get a call to add 'the' before project in both places.

While we're here, the two spaces after the period are also copied from pack. I believe this is against modern style guidelines. Should I make it one space?

Reply:
I vote for yes on the one space. That seems more like a typo to me than I conscious decision. No worries on `the`.

Reply:
Yes, one space (https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#dont-be-spacey)

I can do the more thorough review when I get to #5737. There are other things that bother me in addition to the missing the. For example, 1st sentence is prescriptive, 2nd sentence is 3rd person. It's almost like you're telling the reader what to do instead of what the option does. 

The implies --no-restore: just to confirm, does this mean that you also set --no-restore when passing --no-build?

Reply:
It means that --no-restore is implied by --no-build. There will be no restore if you pass --no-build even you don't explicitly pass --no-restore.

Reply:
I'll add "the" to both pack and publish and remove the extra space in both places. I'll leave it to #5737 to make further edits. Sound good?

Reply:
Perfect @nguerrera!